pkg/controller/chaosscheduler/createEngineForRepeat.go

```diff
@@ -155,27 +171,45 @@ func getRecentUnmetScheduleTime(cs *chaosTypes.SchedulerInfo, cronString string)
 
 func scheduleRepeat(cs *chaosTypes.SchedulerInfo) (string, time.Duration, error) {
 
-	interval, err := fetchInterval(cs.Instance.Spec.Schedule.Repeat.MinChaosInterval)
+	interval, err := fetchInterval(cs.Instance.Spec.Schedule.Repeat.Properties.MinChaosInterval)
 	if err != nil {
 		return "", time.Duration(0), errors.New("error in parsing minChaosInterval(make sure to include 'm' or 'h' suffix for minutes and hours respectively)")
 	}
-	instances, err := fetchInstances(cs.Instance.Spec.Schedule.Repeat.InstanceCount)
+	instances, err := fetchInstances(cs.Instance.Spec.Schedule.Repeat.Properties.InstanceCount)
```

Comment:
Lets set total instances in the status based on what is derived from range & minchaos interval. Won't be possible to honor both (take instance of 20 min interval in a 1 hr range w/ 6 instances)

Reply:
Removing the `instances` field from spec altogether. Will just keep the `runInstances` field in status